src/data/repository/campaign/core/CampaignLogOutcomeRepository.ts

```diff
@@ -6,78 +6,78 @@ export class CampaignLogOutcomeRepository implements OutcomeRepository {
   private outcomes: Outcome[] = [
     {
       id: "a-couriers-job",
-      name: "A courier's job",
       effect: "",
+      translationKey: "outcome.a-couriers-job",
     },
     {
       id: "adamant",
-      name: "Adamant",
       effect: "",
+      translationKey: "outcome.adamant",
     },
     {
       id: "a-hideous-act",
-      name: "A hideous act",
       effect: "",
+      translationKey: "outcome.a-hideous-act",
     },
     {
       id: "a-lovers-request",
-      name: "A lover's request",
       effect: "",
+      translationKey: "outcome.a-lovers-request",
     },
     {
       id: "corrupted-hero",
-      name: "Corrupted hero",
       effect: "",
+      translationKey: "outcome.corrupted-hero",
     },
     {
       id: "curse-breaker",
-      name: "Curse breaker",
       effect: "",
+      translationKey: "outcome.curse-breaker",
     },
     {
       id: "dynamic-duo",
-      name: "Dynamic duo",
       effect: "",
+      translationKey: "outcome.dynamic-duo",
     },
     {
       id: "gentle-dismissal",
-      name: "Gentle dismissal",
       effect: "",
+      translationKey: "outcome.gentle-dismissal",
```

Comment:
For consistency it should be: translation_key

Reply:
oh right, missed that one. thanks.